.github/workflows/lint.yml

```diff
@@ -1,6 +1,4 @@
-on:
-  push:
-  pull_request:
+on: [push,pull_request]
```

Comment:
these still don't specify the main branch, have a look at other examples

Reply:
Lint.yml has been updated to specify the main branch